submissions/mustbefail/tiny-js-world/index.js

```diff
@@ -0,0 +1,89 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const cat = {
+  species: 'cat',
+  name: 'Prokhor',
+  hands: 0,
```

Comment:
Objects of species that do not have hands by natural design do not consequently have hands or any equivalent property

Reply:
Removed property hands